DataFormats/L1TrackTrigger/interface/TTTypes.h

```diff
@@ -36,14 +39,33 @@ using TTTrackPtrT = edm::Ptr<TTTrack<T> >;
 /// Specialized aliases
 typedef edm::Ref<edm::DetSetVector<Phase2TrackerDigi>, Phase2TrackerDigi> Ref_Phase2TrackerDigi_;
 
-typedef edmNew::DetSet<TTStub<Ref_Phase2TrackerDigi_> > TTStubDetSet;
+typedef edmNew::DetSetVector<TTCluster<Ref_Phase2TrackerDigi_> > TTClusterDetSetVec;
```

Comment:
There seems to be git problems here. Hopefully this is a one off error. Your new code is deleting improvements made by other people and restoring older versions of the code. e.g. The definitions of TTClusterDetSetVec, TTStubDetSetVec, TTClusterRef & TTStubRef are no longer using the more compact templated aliases.

Reply:
fixed